components/nds-filters/src/FilterSummary.js

```diff
@@ -154,21 +145,15 @@ const FilterType = PropTypes.shape({
 	className: PropTypes.string
 });
 
-const SortingType =
-	// FilterType +
-	PropTypes.shape({
-		label: PropTypes.string.isRequired,
-		destination: PropTypes.string,
-		onClick: PropTypes.func,
-		elementType: PropTypes.elementType,
-		method: PropTypes.string,
-		className: PropTypes.string,
-		active: PropTypes.bool
-	});
+const SortingType = PropTypes.shape({
+	label: PropTypes.string.isRequired,
+	value: PropTypes.string,
+	active: PropTypes.bool,
+	callback: PropTypes.func
```

Comment:
Can we call `callback` something like `onSelected`  or something similar that's a bit more 'reacty'?